config/environments/test.rb

```diff
@@ -14,6 +14,6 @@
   config.action_controller.allow_forgery_protection    = false
 
   config.action_mailer.delivery_method = :test
-  config.action_mailer.default_url_options = { host: 'localhost:3000' }
+  config.action_mailer.default_url_options = { host: 'localhost:52662' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.